lib/puppet_forge/v3/release.rb

```diff
@@ -38,6 +39,28 @@ def download(path)
         end
       end
 
+      # Uploads the tarbarll to the forge
+      #
+      # @param path [Pathname] tarball file path
+      # @return resp
+      def self.upload(path)
+        encoded_string = Base64.encode64(File.open(path, 'rb').read)
+        data = { file: encoded_string }
+
+        resp = conn.post do |req|
+          req.url '/v3/releases'
+          req.headers['Content-Type'] = 'application/json'
+          req.body = data.to_json
+        end
+
+        [self, resp]
+      rescue Faraday::ClientError => e
+        raise PuppetForge::ReleaseForbidden.from_response(e.response) if e.response && e.response[:status] == 403
```

Comment:
This might read more easily?
```suggestion
        if e.response
            case e.response[:status]
            when 403
                raise PuppetForge::ReleaseForbidden.from_response(e.response)
            when 400
                raise PuppetForge::ReleaseBadContent.from_response(e.response)
            end
        end
```

Reply:
Nice thank you. I prefer that.